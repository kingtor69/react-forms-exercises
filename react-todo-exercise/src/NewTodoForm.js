import React, {useState} from 'react';

const NewTodoForm = () => {
  return (
    <div className="NewTodoForm">

    </div>
  )
}

export default NewTodoForm;
