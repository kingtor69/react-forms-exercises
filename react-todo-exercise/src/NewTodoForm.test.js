import { render } from '@testing-library/react';
import NewTodoForm from './NewTodoForm';

it('renders without crashing', () => {
  render(<NewTodoForm />);
});

it('renders snapshot', () => {
  const { asFragment } = render(<NewTodoForm />);
  expect(asFragment()).toMatchSnapshot();
});
