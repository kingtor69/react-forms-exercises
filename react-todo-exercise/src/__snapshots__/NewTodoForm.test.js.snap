// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<DocumentFragment>
  <div
    class="NewTodoForm"
  />
</DocumentFragment>
`;
